### PR TITLE
Fix NewsOverlay running request on startup

### LINKED ISSUE #9760: News overlay doing web request before requested

### PR DESCRIPTION
**Pull request by EVAST9919**
Same as for DashboardOverlay. Probably should be a thing for every online overlay in the future.

- Closes #9760

### CODE REVIEW

File: osu.Game/Overlays/NewsOverlay.cs [hunk 1]
```diff
@@ -67,7 +67,26 @@ private void load()
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            article.BindValueChanged(onArticleChanged, true);
+            article.BindValueChanged(onArticleChanged);
```
Comment (frenzibyte) [inline osu.Game/Overlays/NewsOverlay.cs:70]: should probably be renamed to `updateDisplay(), then `PopIn` would call it. better than article change trigger.

File: osu.Game/Overlays/NewsOverlay.cs [hunk 2]
```diff
@@ -67,7 +67,26 @@ private void load()
...
+            {
+                article.TriggerChange();
+                displayUpdateRequired = false;
+            }
+        }
+
+        protected override void PopOutComplete()
+        {
+            base.PopOutComplete();
+            displayUpdateRequired = true;
```
Comment (peppy) [inline osu.Game/Overlays/NewsOverlay.cs:89]: this going to trigger a reload every time? is that intentional?

Comment (EVAST9919) [reply osu.Game/Overlays/NewsOverlay.cs:89]: Yes, because we always want to see latest news every time we open the overlay

Comment (peppy) [reply osu.Game/Overlays/NewsOverlay.cs:89]: We don't do this in other overlays, is there any reason to only do it here? And with no limit on how often? If someone spams the button it's going to send tens of requests per second.

I'd suggest not adding such behaviour until we have a refresh button, or some kind of debounce logic.

Comment (Joehuu) [reply osu.Game/Overlays/NewsOverlay.cs:89]: https://github.com/ppy/osu/blob/8a9036938f30da45e3fa37b3446a636abad45cec/osu.Game/Overlays/DashboardOverlay.cs#L96

You can't spam the button as the overlay has to close completely (i.e. pop out complete).

Comment (EVAST9919) [reply osu.Game/Overlays/NewsOverlay.cs:89]: And also we do the same thing for friends overlay

Comment (peppy) [reply osu.Game/Overlays/NewsOverlay.cs:89]: Fair enough.

**Review by peppy — APPROVED** [review]

**Pull request merged**